packages/ui/components/ui/dutch-auction-component/get-progress.ts

```diff
@@ -0,0 +1,16 @@
+const clampToDecimal = (value: number) => Math.min(Math.max(value, 0), 1);
```

Comment:
What does clamp mean here? Maybe a code comment here? (same in other usage too)

Reply:
[Clamping](https://en.wikipedia.org/wiki/Clamping_(graphics)) refers to limiting a value between a given min and max. Will add a comment.